### ISSUE TITLE
Weird issue with an item's display name

### ISSUE THREAD

**Issue by bloggy** [bensku/Skript#234]
I have posted the issue here:
https://forums.skunity.com/t/how-to-get-the-name-of-an-item-which-is-in-a-variable/10938/6?u=pepper82

Skript is unable to show the display name of an item-variable when the item-variable is set to clicked item in an on inventory click event.

Example: I have set a slot in a gui to an obsidian with name &2My-Obsidian

```
on inventory click:
    set {_test} to clicked item
    broadcast "%{_test}'s display name%"
```
result is `"<none>"`

But direct output of the expression works:

```
on inventory click:
    set {_test} to clicked item
    broadcast "%clicked item's display name%"
```
result is `&2My-Obsidian`

I have the following addons installed:

- RandomSK
- SkQuery
- Skellett
- TuSke
- skStuff
- skRayFall

And I am using Skript v2.2-dev20 (because the latest version throws many errors)

**Comment by xXAndrew28Xx** [comment]
Is {_test} <none> too?

**Comment by bensku** [comment]
I wonder if this has something to do with {_test} not being parsed as item... Strange.

**Comment by TheBentoBox** [comment]
I want to note that `clicked item` isn't a vanilla Skript expression, so this may very well be an issue with that expression in those addons. I won't close this until you verify though Bensku.

**Comment by JeepSmash** [comment]
I have had similar issues with this when using SkQuery's gui commands.  I concluded that the issue was the slot item being returned was for the players inventory and NOT the gui inventory.  Try filling your player inventory with custom named items then open your GUI and click on a slot in the GUI and I found that the value returned for "clicked item" is an item in the players inventory.  This was not an issue with Skript but with the addon i was using at the time SkQuery. 

Breakdown of what I figured out was going on:
1 player opens fake chest inventory as GUI (using SkQuery)
2 player clicks on GUI slot
3 SkQuery reads the clicked slot's number
4 SkQuery returns the slot's number
5 the running script requests the clicked items info
6 SkQuery looks up the item in players inventory **(instead of the fake inventory)** based on the returned slot number
